### PR TITLE
All files are added

### PR DESCRIPTION
**Pull request by asadmisrani**
Hello, Microverse it is a personal portfolio site about Portfolio: a finished mobile version made for Education purposes, it is used with Figma design to create a UI, Flexbox to place elements on the page, and CSS Grid to place elements.

1- **There are no linter errors:**
2- **Used correct GitHub Flow:**
3- **Documented in a professional way:**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+      <ul>
+        <li>
+          <a target="_blank" href="#"><img src="assets/twitter.png"
+              alt="Twitter" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/Linkedin icon.png"
+              alt="LinkedIn" /></a>
+        </li>
+        <li>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use the semantics tag for accessibility on your page. You may consider grabbing all the introduction part in a `section` tag and all the projects in one `section`. 
You do not need to grab all small parts in a section.
`Hint: You may consider checking this` [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) `to know more about where you can use the section tag.`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+          <a target="_blank" href="#"><img src="assets/angellist.png"
+              alt="GitHub" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <section class='button-down'>
+      <a target="_blank" href="#"><img src="assets/disabled.png" alt="down" /></a>
+    </section>
```
Comment (christianbiring1) [inline index.html]: - [ ] Remove these `section` tags around here since they don't have any semantic meaning.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  </section>
+
+  <section class="projects">
+    <section class='recent-works'>
+      <h1 class="my-recent-works">My Recent Works</h1>
+      <hr class="hr-works">
+    </section>
```
Comment (christianbiring1) [inline index.html]: - [ ] Remove these `section` tags around here since they don't have any semantic meaning.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+        <h2>Languages</h2>
+        <ul>
+          <li>JavaScript</li>
+          <li>Python</li>
+          <li>Html</li>
+          <li>CSS</li>
+        </ul>
+
+      </div>
+      <div class="frameworks">
```
Comment (christianbiring1) [inline index.html]: -  [ ] As suggested early, Kindly make sure all the about part is grabbed in one `section` tag

File: style.css [hunk 5]
```diff
@@ -0,0 +1,454 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+
+* {
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure your page fits size of the screen. The user does not need to scroll to the right to see the entirety of the page.
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @asadmisrani , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` :+1: 
- [ ] Good UI :art: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (asadmisrani) [reply index.html]: @christianbiring1
I update the section to div in line number 23 to +46.

File: index.html (hunk 2 above)
Comment (asadmisrani) [reply index.html]: @christianbiring1 
I updated the section tag from +60 to +62.

File: index.html (hunk 3 above)
Comment (asadmisrani) [reply index.html]: @christianbiring1 
I updated the section tag from +64 to +69.

File: index.html (hunk 4 above)
Comment (asadmisrani) [reply index.html]: @asadmisrani 
I updated the section tag from 175 to +198.

File: style.css (hunk 5 above)
Comment (asadmisrani) [reply style.css]: @christianbiring1 
I updated the entire page fit the size of the screen.

File: index.html (hunk 1 above)
Comment (asadmisrani) [reply index.html]: @christianbiring1 
I updated the section tag from +23 to +46.

**Review by amejid — APPROVED** [review]
Hi @asadmisrani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
Thank you very much

**Pull request merged**